index.html

```diff
@@ -175,7 +175,11 @@ <h3 class="key-speakers-text">Key Speakers</h3>
             </div>
             <i class="fa-solid fa-horizontal-rule line-red-main"></i>
         </div>
-        <div id="speakers-container"></div>
+        <div id="speakers-container" class="container-speaker"></div>
+        <div class="see-more-bar">
```

Comment:
as suggested by the previous code review

> From the project requirements, you should stick to [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), But as you can see from [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) in the home page on the mobile screen instead of the partner's section and the footer section we have the more button so please make sure that you work on that.

Your Design 
![image](https://user-images.githubusercontent.com/60612329/226144544-d83811a9-b59d-42b8-bb23-3637357a1842.png)

required design
![image](https://user-images.githubusercontent.com/60612329/226144534-52ef301e-7f24-4f34-aecd-400621e0069c.png)